dvc/repo/push.py

```diff
@@ -43,20 +43,18 @@ def push(
     )
 
     dolt_pushed = 0
-    remote_conf = None
-    for t in expanded_targets:
-        if os.path.exists(os.path.join(t, ".dolt")):
-            if not remote_conf:
+    if expanded_targets is not None:
+        remote_conf = None
+        for t in expanded_targets:
+            if os.path.exists(os.path.join(t, ".dolt")):
                 remotes = self.config.get("remote", None)
                 if not remotes:
                     break
                 remote_conf = remotes.get(remote, None)
                 if not remote_conf:
                     break
-            db = dolt.Dolt(t)
-            remote_url = remote_conf.get("url")
-            db.remote(name=remote, url="file://" + remote_url, add=True)
-            db.push(remote=remote, set_upstream=True, refspec="master")
-            dolt_pushed += 1
```

Comment:
there's nothing wrong with independently adding a dolt remote, but i think this breaks the behavior i expect in my tests because you'd have to work outside of the DVC interface

Reply:
I see, this was a workaround for an error when the remote already existed. If I understand correctly, this has since been fixed in [this](https://github.com/dolthub/dvc/commit/7a2001630d8ab9db1a6ec5b508d169ea19ae3b7e#diff-122748fe1e59843b77d6226e86d471fde21996d784c09e5a0f2c2ed9edeb2117) commit, outside of the PR.